utopia-remix/app/util/errors.ts

```diff
@@ -22,3 +22,8 @@ export function isErrorWithStatus(u: unknown): u is ErrorWithStatus {
   const maybe = u as ErrorWithStatus
   return maybe.status != null && maybe.statusText != null && maybe.data != null
 }
+
+export function isLikeApiError(u: unknown): u is ApiError {
+  const maybe = u as ApiError
+  return maybe.status != null && maybe.message != null
```

Comment:
We should be checking that this is actually not null and an object before checking the fields, like we do here <https://github.com/concrete-utopia/utopia/blob/master/editor/src/components/canvas/canvas-component-entry.tsx#L154>

Reply:
[`d199a17` (#4979)](https://github.com/concrete-utopia/utopia/pull/4979/commits/d199a173ecaf1db12f9c92177c73081d795fad8d)

Reply:
also done it for another helper